docs/requirements/hardware-proposal.md

```diff
@@ -0,0 +1,110 @@
+# Hardware proposal
+
+Current document covers [requirements](requirements.md) in proposal real hardware
+solutions, which meet requirements gathered earlier.
+
+## Infineon CY8C6xxxx Series
+
+* Link to documentation: [Infineon PSoC6 Family](https://www.infineon.com/cms/en/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/psoc-6-32-bit-arm-cortex-m4-mcu/)
+* Point of interest: PSoC 64-Secured MCU series
+* 32-bit Programmable System-on-Chip, integrates in one structure
+ARM Cortex M4 (150MHz) with Cortex M0+, 100MHz
+* Various variants of memories:
+  * flash 384-1856 KB
+  * RAM 176-944KB
+* Interfaces I2C, SPI, UART, USB
+* Crypto Accelerator (DES/TDES, AES, SHA, CRC, TRNG, RSA/ECC)
+* Software ModusToolbox: [Modus Toolbox](https://www.infineon.com/cms/en/design-support/tools/sdk/modustoolbox-software/)
+
+## SparkFun QuickLogic Thing Plus - EOS S3
+
+* Link to documentation: [QuickLogic Thing Plus - EOS S3 Datasheet](https://www.sparkfun.com/products/17273)
+* eFPGA-enabled ARM Cortex©-M4F MCU
+* Based fully on open source hardware
+* Supports open-source FPGA synthesis software - SymbiFlow (now known as F4PGA).
+  * Partial upstream support from F4PGA - lack of BlockRAM support can be a
+    serious obstacle for complex FPGA designs. This is not a problem for a
+    simple LPC controller.
+* Downstream support for [Zephyr](https://github.com/QuickLogic-Corp/zephyr/tree/eos-s3-support)
+  and FreeRTOS (as part of the [QORC SDK](https://github.com/QuickLogic-Corp/qorc-sdk))
+  * Based on an old version of Zephyr. Mainline Zephyr has only the most basic
+    support EOS S3 and it is not usable without porting additional drivers.
+
+## STM32L476RG Nucleo board
+
+* Link to documentation: [STM32L476RG Nucleo Web Page](https://www.st.com/en/evaluation-tools/nucleo-l476rg.html)
+* Link to datasheet: [STM32L476RG Nucleo Datasheet](https://www.st.com/resource/en/datasheet/stm32l476rg.pdf)
+* Ultra low power ARM Cortex-M4 MCU with FPU
+* Power supply 1.71-3.6V
+* 80MHz system clock
+* 3xSPI interfaces
+* Cryptography engine
+* SPI interface frequency, below citation from datasheet:
+"Three SPI interfaces allow communication (...) up to 24 Mbits/s
+slave modes".
+
+## STM32G474 Nucleo board
+
+* Link to documentation: [STM32G474 Nucleo Board Documentation](https://www.st.com/en/evaluation-tools/nucleo-g474re.html)
+* ARM Cortex-M4 MCU
+* Power supply 1.71-3.6V
+* 170MHz system clock
+* 4xSPI interfaces, 4-16 programmable bit frames
+  * Max SPI clock frequency:
+    * Slave, receiver mode,  1.71V < Vdd < 3.6V: 50MHz
+    * Slave mode transmitter, with full duplex, 2.7V < Vdd < 3.6V: 41MHz
+    * Slave mode Transmitter, with full duplex, 1.7V < Vdd < 3.6V: 27MHz
+* Mathematical hardware accelerators
+  * CORDIC for trigonometric functions acceleration
+  * FMAC: filter mathematical accelerator
+
+Excluded chips
```

Comment:
@3mdeb-karolzmyslowski Thius shoul be a section maybe? Have you seen how bad it looks after rendering?

More to that, if we have a section of `excluded chips`, we should also have a section on `approved chips`, perhaps?

Reply:
(info: corrected, will be visible after commit, push)